docs/docs/tutorials/streaming/index.md

```diff
@@ -393,3 +393,57 @@ Final output:  Prediction(
     sum='9'
 )
 ```
+
+## Synchronous Streaming
+
+By default calling a streamified DSPy program produces an async generator. In order to get back
+a sync generator, you can set the flag `async_streaming=False`:
+
+```
```

Comment:
```suggestion
```python
```

Reply:
Actually is this block necessary? It seems to be duplicated with the following example.

Reply:
ah thank you for catching it